starter_code/src/movies.js

```diff
@@ -76,6 +81,18 @@ function ratesAverage(arr) {
 }
 
 // Iteration 5: Drama movies - Get the average of Drama Movies
+function dramaMoviesRate(arr) {
+  const arrDram = arr.filter(e => {
+    return e.genre.includes("Drama");
+  });
+
+  let rateTotal = arrDram.reduce((sum, movie) => {
+    return sum + movie.rate;
+  }, 0);
+
+  return rateTotal / arrDram.length;
+}
```

Comment:
same than previous iteration, you round the number return to 2 decimals with the .toFixed() method, and you convert the whole thing to a Number type:
return Number((rateTotal / arrDram.length).toFixed(2));